From 177e691db7ef6278b3210e452c3dee3bd920f0d0 Mon Sep 17 00:00:00 2001
From: Chris Burr <chrisburr@users.noreply.github.com>
Date: Tue, 4 Dec 2018 18:35:30 +0100
Subject: [PATCH] Fix macro check to support clang 4.0.0

__cplusplus is defined to be 201406L in clang 4.0.0 for cxx17.
Relax the check to be one month later than the cxx14 value (201402L).

[1] https://github.com/llvm-mirror/clang/blob/release_40/lib/Frontend/InitPreprocessor.cpp#L379
---
 core/foundation/test/testNotFn.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/foundation/test/testNotFn.cxx b/core/foundation/test/testNotFn.cxx
index 601aaba98b6..2d6d4c7f3fd 100644
--- a/core/foundation/test/testNotFn.cxx
+++ b/core/foundation/test/testNotFn.cxx
@@ -1,4 +1,4 @@
-#if __cplusplus < 201703L && !defined(_MSC_VER)
+#if __cplusplus < 201403L && !defined(_MSC_VER)
 
 #include "ROOT/RNotFn.hxx"
 
